modules/ossm-release-2-4-11.adoc

```diff
@@ -0,0 +1,34 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-4-11_{context}"]
+= {SMProductName} version 2.4.11
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs) and is supported on {product-title} 4.14 and later.
+
+[id=ossm-release-2-4-11-components_{context}]
+== Component updates
+
+|===
+|Component |Version
+
+|Istio
+|1.16.7
+
+|Envoy Proxy
+|1.24.12
+
+|Kiali Server
+|1.65.15
+|===
+
+[id="ossm-fixed-issues-2-4-11_{context}"]
+== Fixed issues
```

Comment:
Fixed issue, singular, since there is only 1.

Reply:
Thanks for pointing this out. Also changed **New features** to **New feature** in 2.6.2, since there's only 1.